spark/data-science-base/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM pubnative/pyspark-ci:base-e849b23
```

Comment:
please add `# pubnative/pyspark-ci:version-tag`

Reply:
@bruwozniak @miry we end up with `# pubnative/pyspark-ci:data-science-base-${COMMIT}` comment